test/generated/testproto/grpc/dummy_pb2_grpc.pyi

```diff
@@ -7,70 +7,159 @@ import abc
 import collections.abc
 import grpc
 import grpc.aio
+import sys
 import testproto.grpc.dummy_pb2
 import typing
 
+if sys.version_info >= (3, 13):
+    import typing as typing_extensions
+else:
+    import typing_extensions
+
 _T = typing.TypeVar("_T")
 
 class _MaybeAsyncIterator(collections.abc.AsyncIterator[_T], collections.abc.Iterator[_T], metaclass=abc.ABCMeta): ...
 
 class _ServicerContext(grpc.ServicerContext, grpc.aio.ServicerContext):  # type: ignore[misc, type-arg]
     ...
 
-class DummyServiceStub:
-    """DummyService"""
-
-    def __init__(self, channel: typing.Union[grpc.Channel, grpc.aio.Channel]) -> None: ...
-    UnaryUnary: grpc.UnaryUnaryMultiCallable[
+_MTVDummyService0 = typing_extensions.TypeVar(
```

Comment:
don't love the name here. Is there a way to make the naming based on the service name / method name - rather than these iterating numbers?

Perhaps instead of {i} use the {method.name}

`MTV` also feels rather opaque. "MethodTypeVar" isn't particularly useful to the enduser (who will probably see these types when hovering around in vscode).

Perhaps we don't need the "_MTV" prefix at all?

Reply:
It's been a while so I'm going to have to remember why I did the naming like that. I know I wasn't a huge fan of it, but wanted to get something down on paper. Let me see what it looks like to not have the prefix.

Reply:
Ok here we go. I had to keep the leading underscore because of stubtest (left a comment with the SO explaining it, couldn't actually find it in stubtest docs, didn't look too long). So now it is just `_ServiceNameMethodNameType` which shouldn't conflict. Also cleaned it up so the typevar name building so it is more maintainable.